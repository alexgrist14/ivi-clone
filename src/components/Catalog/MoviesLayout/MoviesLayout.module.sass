.layout
  display: grid
  gap: 15px
  margin-top: 24px
