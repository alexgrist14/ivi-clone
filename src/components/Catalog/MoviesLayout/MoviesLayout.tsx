import { FC, ReactNode } from "react";
import styles from "./MoviesLayout.module.sass";
import Container from "/src/UI/Container/Container";

interface MoviesLayoutProps {
  children: ReactNode;
}

const MoviesLayout: FC<MoviesLayoutProps> = ({ children }) => {
  return (
    <Container>
      <div className={styles.layout}>{children}</div>
    </Container>
  );
};
export default MoviesLayout;
