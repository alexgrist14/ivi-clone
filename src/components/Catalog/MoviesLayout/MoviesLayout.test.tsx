import { render } from "@testing-library/react";
import MoviesLayout from "./MoviesLayout";

describe("Movies layout component", () => {
  test("layout rendered correctly", () => {
    const { container } = render(<MoviesLayout>test</MoviesLayout>);

    expect(container).toMatchSnapshot();
  });
});
