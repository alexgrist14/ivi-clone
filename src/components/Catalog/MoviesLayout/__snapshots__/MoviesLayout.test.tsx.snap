// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movies layout component layout rendered correctly 1`] = `
<div>
  <div
    class="container"
    data-testid="container"
  >
    <div
      class="layout"
    >
      test
    </div>
  </div>
</div>
`;
